sklearn/linear_model/_stochastic_gradient.py

```diff
@@ -1494,9 +1525,19 @@ class SGDRegressor(BaseSGDRegressor):
     average_coef_ : ndarray of shape (n_features,)
         Averaged weights assigned to the features.
 
+        .. deprecated:: 0.23
+            Attribute ``average_coef_`` was renamed to
+            ``_average_coef_`` in version 0.23 and will
+            be removed in 0.25.
+
     average_intercept_ : ndarray of shape (1,)
         The averaged intercept term.
 
+        .. deprecated:: 0.23
+            Attribute ``average_intercept_`` was renamed to
+            ``_average_intercept_`` in version 0.23 and will
+            be removed in 0.25.
```

Comment:
We don't generally document private attributes, so I would say ,

> Attribute ``average_intercept_`` was deprecated in version 0.23 and will
> be removed in 0.25. 

Indicating the private attribute to which it was renamed in the warning should be enough IMO.